docs/docs/multiplayer/time.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 61
+---
+
+# Time
+
+## Game time
+
+If your game requires access to time, Rune exposes `Rune.gameTimeInSeconds()` that you can use inside logic.js.
+Calling it returns the amount of seconds that have passed since the start of the game.
+
+At the moment Rune allows timers with an accuracy of a second.
```

Comment:
```suggestion
Rune exposes `Rune.gameTimeInSeconds()` that you can use inside logic.js, which returns the amount of seconds that have passed since the start of the game. At the moment Rune provides time precision of a second, which should work well for most casual game purposes.
```